salt/modules/cmdmod.py

```diff
@@ -378,9 +378,6 @@ def _run(cmd,
             msg = 'missing salt/utils/win_runas.py'
             raise CommandExecutionError(msg)
 
-        if not isinstance(cmd, list):
-            cmd = salt.utils.shlex_split(cmd, posix=False)
-
         cmd = ' '.join(cmd)
```

Comment:
```
cmd = ' '.join(cmd)
```
Notice this after the merge
Should be
```
if if isinstance(cmd, list):
    cmd = ' '.join(cmd)
```

Reply:
Good catch, see https://github.com/saltstack/salt/pull/43869.

I thought of maybe using `salt.ext.six.moves.shlex_quote()` on each element, but was annoyed to find out that it uses single quotes on Windows. This would break things like the `find` command in Windows which expects the string to be double-quoted. Don't know if you have a better solution, other than just not passing the command as a list in those cases.